### DISCUSSION TITLE
What blog post or guide would you like to read?

### DISCUSSION THREAD

**Discussion by samalba** [dagger/dagger#2761] [General]
Is there a specific use case, or blog post about a specific deployment example, or even new docs guides that you would like to read? Anything that can help you build your own dagger Plan?

**Comment by sagikazarmark** [comment]
I'd love to read about integrating Dagger into a Jenkins pipeline. It's something I'll have to do at some point and would be nice to have some reading material by then.

**Reply by jpadams** [reply]
@sagikazarmark what aspects would you want covered?
We have a basic Jenkinsfile example and an accompanying video here: https://docs.dagger.io/1201/ci-environment (Jenkins tab).

I've heard other users mention:
- Jenkins shared libraries
- Generating pipelines from shared libraries

**Reply by sagikazarmark** [reply]
Ah, haven't seen that, thanks!

We do have a shared jenkins library with that accepts declarative configuration to build the final, multi-stage pipeline.

The library takes care of things like fetching secrets from a Hashicorp Vault and injecting them as env vars. Seeing how that would work with Dagger would be interesting.

Otherwise having at least a basic pipeline is a good start, so thanks for pointing me to that.

**Reply by jpadams** [reply]
> We do have a shared jenkins library with that accepts declarative configuration to build the final, multi-stage pipeline.

I'd love to learn more about that! Any pointers?

> The library takes care of things like fetching secrets from a Hashicorp Vault and injecting them as env vars. Seeing how that would work with Dagger would be interesting.

Very cool! I'd love to get something like that going as an example, for sure. Any suggestions/pointers/links welcome :)

**Reply by sagikazarmark** [reply]
> I'd love to learn more about that! Any pointers?

Sadly it's not open source, but it's basically a set of groovy scripts.

I want to integrate Dagger into our pipeline at some point. I'll see if I can publish some of it once it's done.
